src/Components/Shared/Button/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from './button.module.css';
+
+const Button = ({ text, type, clickBtn }) => {
```

Comment:
Here onClick is easiest to read
```suggestion
const Button = ({ text, type, onClick }) => {
```